server/routes/overview/overviewController.ts

```diff
@@ -1,92 +1,26 @@
-import createError from 'http-errors'
 import { RequestHandler } from 'express'
-import type { Goal, PrisonerGoals } from 'viewModels'
-import { EducationAndWorkPlanService, InductionService } from '../../services'
-import { mostRecentFunctionalSkills } from '../functionalSkillsResolver'
-import logger from '../../../logger'
+import { EducationAndWorkPlanService } from '../../services'
 import OverviewView from './overviewView'
 
 export default class OverviewController {
-  constructor(
-    private readonly inductionService: InductionService,
-    private readonly educationAndWorkPlanService: EducationAndWorkPlanService,
-  ) {}
+  constructor(private readonly educationAndWorkPlanService: EducationAndWorkPlanService) {}
 
   getOverviewView: RequestHandler = async (req, res, next): Promise<void> => {
     const { prisonNumber } = req.params
-    const {
+    const { prisonerSummary, curiousInPrisonCourses, actionPlanReviews, prisonerFunctionalSkills, induction } =
+      res.locals
+
+    const prisonerGoals = await this.educationAndWorkPlanService.getAllGoalsForPrisoner(prisonNumber, req.user.username)
+
+    const view = new OverviewView(
       prisonerSummary,
-      showServiceOnboardingBanner,
+      prisonerFunctionalSkills,
       curiousInPrisonCourses,
       actionPlanReviews,
-      prisonerFunctionalSkills,
-    } = res.locals
-
-    try {
-      const [inductionExists, prisonerGoals] = await Promise.all([
-        this.inductionService.inductionExists(prisonNumber, req.user.username),
-        this.educationAndWorkPlanService.getAllGoalsForPrisoner(prisonNumber, req.user.username),
-      ])
-
-      const isPostInduction = Boolean(inductionExists)
-      const functionalSkills = mostRecentFunctionalSkills(prisonerFunctionalSkills)
```

Comment:
And all this stuff about working out the counts etc has all been moved into the View class below (`OverviewView.ts`) which makes the controller so much simpler 👍